### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Matthew-Marsh**
Hey @CamA-JCU, 

Just mentioning you in this pull request so you can review my code work for practical 6. 

Thanks,
Matt

### CODE REVIEW

File: prac_06/guitar.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        """ Print guitar details """
+        return "{} ({}) : ${:.2f}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        """ Determine the guitar age """
+        return Guitar.CURRENT_YEAR - self.year
+
+    def is_vintage(self):
+        """ Determine if guitar is vintage """
+        return self.get_age() > 50
```
Comment (CamA-JCU) [inline prac_06/guitar.py:25]: Should this be greater than or equals to so that 50 is included?

File: prac_06/programming_language.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+""" Class will take in programming language and its attributes to process """
+
+class ProgrammingLanguage:
+    """ Programming language and its attributes  """
+
+    def __init__(self, name, typing, reflection, year):
```
Comment (CamA-JCU) [inline prac_06/programming_language.py:6]: Should you have default attributes?

File: prac_06/guitar.py (hunk 1 above)
Comment (Matthew-Marsh) [reply prac_06/guitar.py:25]: Probably should be greater than and equals to. Thanks.

File: prac_06/programming_language.py (hunk 2 above)
Comment (Matthew-Marsh) [reply prac_06/programming_language.py]: It could. So i added some defaults.

**Pull request merged**